driver-core/src/main/com/mongodb/client/model/expressions/Expressions.java

```diff
@@ -136,4 +136,20 @@ public static <R extends Expression> R ofNull() {
         return new MqlExpression<>((cr) -> new AstPlaceholder(new BsonNull()))
                 .assertImplementsAllExpressions();
     }
+
+    static NumberExpression numberToExpression(final Number number) {
```

Comment:
I don't think something prevents us from naming this method `of`. Am I missing something?

Reply:
I just thought the name was more descriptive, since there was no constraint (as with `of`) of being as short as possible.

Reply:
I have realized that this method is not part of public API. If I noticed that right away, I would not have asked about renaming.